shm/Player.cpp

```diff
@@ -1 +1,11 @@
 #include "Player.hpp"
+
+size_t countAvailableSpace() {
+    auto total = std::accumulate(begin(ship_->getAllCargos()), end(ship_->getAllCargos()), 0, [](const auto& cargo) {
+        cargo->getAmount();
```

Comment:
Arrow operator is correct here? When I thinking about it now I should use dot operator.

Reply:
Hard to tell because there is no Ship implementation on our git 😄

Reply:
Actually it is, in module1/solutions/exercise3.cpp but not that much of it 😄  No getAllCargos() function nor cargo object are present there.

But basically it depends how Cargo will be stored in Ship. If by pointer then -> operator is fine.

Reply:
What do you think of changing name "total" to something different like "totalOccupied" or "occupiedSpace"? At first I thought that "total" is whole ship capacity.

Reply:
You're right that total is not the best name for this variable. From your two suggestions I  prefer "occupiedSpace", so I will change on it if everyone else agree.

Reply:
Changed.